geoengine/types.py

```diff
@@ -44,6 +44,24 @@ def __init__(self, spatial_bounds: Tuple[float, float, float, float], time_inter
     def bbox_str(self) -> str:
         return ','.join(map(str, self.__spatial_bounds))
 
+    @property
+    def bbox_ogc(self) -> str:
+        # TODO: properly handle axis order
+        bbox = self.__spatial_bounds
+        if self.__srs == "EPSG:4326":
+            return [bbox[1], bbox[0], bbox[3], bbox[2]]
+        else:
+            return bbox
+
+    @property
+    def resolution_ogc(self) -> Tuple[float, float]:
+        # TODO: properly handle axis order
+        res = self.__resolution
+        if self.__srs == "EPSG:4326":
+            return [-res[1], res[0]]
```

Comment:
why does the sign change?

Reply:
afaik resolution is always positive and here I turn it into pixel step sizes which are negative in y direction for wgs84

Reply:
ok maybe we need to document that somewhere